lumicks/pylake/file.py

```diff
@@ -119,13 +120,20 @@ def print_group(group, name="", indent=-2):
         return print_attributes(self.h5) + "\n" + print_group(self.h5)
 
     def _get_force(self, n, xy):
-        return Continuous.from_dataset(self.h5["Force HF"][f"Force {n}{xy}"], "Force (pN)")
+        force_group = self.h5["Force HF"][f"Force {n}{xy}"]
+        calibration_data = ForceCalibration(ForceCalibration.from_dataset(self.h5, n, xy))
```

Comment:
`ForceCalibration(ForceCalibration.from_dataset(...))` is a bit redundant. You should make sure that `from_dataset()` already returns a `ForceCalibration`.

Reply:
👍